radionets/evaluation/plotting.py

```diff
@@ -288,8 +288,8 @@ def visualize_with_fourier_diff(
     real_truth, imag_truth = img_truth[0], img_truth[1]
 
     if amp_phase:
-        real_pred = 10 ** (10 * np.array(real_pred, dtype="float128") - 10) - 1e-10
-        real_truth = 10 ** (10 * np.array(real_truth, dtype="float128") - 10) - 1e-10
```

Comment:
why is this no problem anymore?

Reply:
I don't remember what the exact problem was. I tested some things after removing this and everything seemed fine. This casting produces a warning which I wanted to remove. MAybe you can remind me, what the problem was.

Reply:
I think, we had some problems with the evaluation, when we did fast implementations during the review process of the paper. I cannot remember the exact problem. We should keep it in mind, but should be fine for the moment.